Flat/src/AnalyzerUtilities.cc

```diff
@@ -215,4 +215,32 @@ bool IsMatched(vector<panda::Particle*>*objects,
   }
   return false;
 }
+bool JetPUID(double pt, double eta, double puid)
```

Comment:
Sorry, this isn't what I meant by cleaning up PUID. I meant, instead of a dozen hardcoded numbers, can we just have a 2D histogram.

Reply:
Yes really. The electron IP cuts are 4 numbers, and even still, I don't particularly like the fact that they are hardcoded. Here's the histogram: http://t3serv001.mit.edu/~snarayan/misc/puid.root

Reply:
You'll just need to add a `cPUID` to the corrections enum, load the histogram as we do all other corrections, and then check whether it passes with `jet.puid > GetCorr(cPUID, abs(jet.eta()), jet.pt())`